src/services/game-logic/pieces/Knight.ts

```diff
@@ -0,0 +1,23 @@
+import { PieceNames } from '../../../enums';
+import { Square } from '../../../models/Square';
+import { Piece } from './Piece';
+
+class Knight extends Piece {
+    name = PieceNames.KNIGHT;
+
+    getPossibleMoves(): Square[] {
+        let possibleRowMoves = [2, 1, -1, -2, -2, -1, 1, 2];
+        let possibleColumnMoves = [1, 2, 2, 1, -1, -2, -2, -1];
+
+        let destination: Square[] = [];
+        let { row, column } = this.position;
```

Comment:
Te zmienne z założenia nie będą się zmieniać, nie dawajmy sobie więc szansy na to by przez przypadek je nadpisać i wykorzystajmy const zamiast let

Reply:
poprawione, dzięki, doktorze

Reply:
na developie dalej jest let x 3 :P